bin/provision.sh

```diff
@@ -315,4 +316,12 @@ sudo chmod 755 phpunit
 cd /var/www/html
 sudo chmod 755 psh.phar
 
+label "setting up translation"
+cd /var/www/html/bin/
+sudo chmod 755 translation
+cd /usr/local/bin && sudo ln -sf /var/www/html/bin/translation
```

Comment:
is it necessary to make it global? I would like the psh.phar way. So we can use it in our deployment scritps on test and live